rs/sns/swap/canister/canister.rs

```diff
@@ -497,7 +403,7 @@ fn encode_metrics(w: &mut ic_metrics_encoder::MetricsEncoder<Vec<u8>>) -> std::i
     )?;
     w.encode_gauge(
         "sale_cycle_balance",
-        dfn_core::api::canister_cycle_balance() as f64,
+        ic_cdk::api::canister_balance() as f64,
```

Comment:
Note that in our security best practices we discourage public access to the cycles balance. https://internetcomputer.org/docs/current/developer-docs/security/security-best-practices/observability-and-monitoring#do-not-publicly-reveal-canisters-cycles-balance